packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -1,4 +1,17 @@
 <template>
+	<toolbar>
+		<h5>Workflow name</h5>
+		<div class="button-group">
+			<Button label="Clean up layout" class="secondary-button" text @click="cleanUpLayout" />
+			<Dropdown
+				icon="pi pi-plus"
+				class="primary-dropdown"
+				option-label="label"
+				:options="contextMenuItems"
+				placeholder="Add component"
+			/>
+		</div>
+	</toolbar>
```

Comment:
There is a slot for this in `<tera-infinite-canvas>`, create a `<template #foreground>` and put this toolbar there.